src/scaffoldmaker/utils/eft_utils.py

```diff
@@ -857,3 +961,72 @@ def determineCubicHermiteSerendipityEft(mesh, nodeParameters, nodeLayouts):
                 deltas[on][ed] = otherElementDerivative
 
     return eft, scalefactors
+
+
+CubicHermiteSerendipityValueLabels = [
+    Node.VALUE_LABEL_VALUE, Node.VALUE_LABEL_D_DS1, Node.VALUE_LABEL_D_DS2, Node.VALUE_LABEL_D_DS3]
```

Comment:
Is there a reason why this variable "CubicHermiteSerendipityValueLabels" is outside the "getTricubicHermiteSerendipityElementNodeParameter" function? Seems like it's only used within the function.

Reply:
To guarantee it is constructed once. It's something that could also be used in multiple functions.